researchstudio-sat/webofneeds
# Feat loadmsgonclick

implements the ability to load messages that are not in the state yet but seem to be visible in a proposal by clicking on the message

blocked by #2134

fixes #2135
 how to test:
1) write quite alot of messages with debugbot (usually send 9 /send 9/usage should suffice)

2) go to the your first message and select propose
3) go to the debugbots first message and select propose as well
4) type propose any 5 and wait for all the proposal by the bot
5) reload the page
6) you should see approx. the last 15msgs now
7) the agreements/proposals might show a "message not loaded"-text
8) click on all of these messages
9) if you have received all the messages the pr is ok... just be sure its proposals of both sent and received msgs
